### DISCUSSION TITLE
New alerts GUI bug

### DISCUSSION THREAD

**Discussion by idolaman** [Security-Onion-Solutions/securityonion#1595] [Unsupported Versions] [answered]
Happens when both the 'Escalate' toggle and 'Acknowledge' toggle are disabled.
When enabling 'Escalate' toggle, the 'Acknowledge' toggle also become enable.

**Comment by jertel** [comment, marked as answer]
This is not a bug, it's operating as designed per a request by other SO team members. The reasoning is that enabling both will ensure that the user sees escalated and acknowledged events. If only the escalate toggle is enabled then the user is unlikely to see any results, since its less common for an event to be escalated but not acknowledged. That scenario only occurs when a user escalates an event but then later "un-acknowledges" the event.
